src/main/java/ui/listpanel/ListPanelCell.java

```diff
@@ -54,6 +52,18 @@ public void updateItem(TurboIssue issue, boolean empty) {
         this.setId(issue.getRepoId() + "_col" + parentPanelIndex + "_" + issue.getId());
     }
 
+    private void updateStyleForClosedIssues(TurboIssue issue){
```

Comment:
Seems this is not just about closed issues (in your `else` clause you change the style of open issues).
Shouldn't it be called something like `updateStyleToMatchStatus`?

The logic should be like this?
if open, apply open style
else, apply closed style.